.circleci/config.yml

```diff
@@ -186,11 +186,17 @@ workflows:
     version: 2
     setup_and_quality:
         when:
-            not: <<pipeline.parameters.nightly>>
+            and:
+                - equal: [<<pipeline.project.git_url>>, https://github.com/huggingface/transformers]
+                - not: <<pipeline.parameters.nightly>>
+        jobs:
+            - fetch_tests
+
+    setup_and_quality_2:
+        when:
+            not:
+                 equal: [<<pipeline.project.git_url>>, https://github.com/huggingface/transformers]
         jobs:
-            - check_circleci_user
-            - check_code_quality
-            - check_repository_consistency
             - fetch_tests:
                 # [reference] https://circleci.com/docs/contexts/
                 context:
```

Comment:
This is for private repositories.

Reply:
Yeah, I think this is kind of what Henna suggested in her message. As external contributors won't have permission to access `TRANSFORMERS_CONTEXT`, we should either remove the `context:` or remove the private contents that users can't access.